x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/empty_state.tsx

```diff
@@ -50,22 +52,26 @@ export const EmptyState: React.FC = () => {
             </p>
           }
           actions={
-            <EuiButtonTo
-              data-test-subj="EmptyStateCreateFirstEngineCta"
-              fill
-              to={ENGINE_CREATION_PATH}
-              onClick={() =>
-                sendAppSearchTelemetry({
-                  action: 'clicked',
-                  metric: 'create_first_engine_button',
-                })
-              }
-            >
-              <FormattedMessage
-                id="xpack.enterpriseSearch.appSearch.emptyState.createFirstEngineCta"
-                defaultMessage="Create an engine"
-              />
-            </EuiButtonTo>
+            <>
+              <EuiButtonTo
+                data-test-subj="EmptyStateCreateFirstEngineCta"
+                fill
+                to={ENGINE_CREATION_PATH}
+                onClick={() =>
+                  sendAppSearchTelemetry({
+                    action: 'clicked',
+                    metric: 'create_first_engine_button',
+                  })
+                }
+              >
+                <FormattedMessage
```

Comment:
In this project we prefer `i18n.translate`. We only use `FormattedMessage` if we need something like nested html.